templates/collections/explorer_index_page.html

```diff
@@ -30,7 +30,11 @@ <h2>{{ page.articles_title }}</h3>
 
         <div class="featured-articles__cta">
             {% comment %} TODO: update BEM button styles for consistency across site {% endcomment %}
-            <a href="/stories" class="tna-button--dark">Browse all stories</a>
+            <a href="/stories"
```

Comment:
We still need to change this so it isn't hard coded. Could we do that as part of this PR, or does it need a separate piece of work?

Reply:
Hi @janehughes we can do it here! 😄

Reply:
As agreed on slack - we'll get this done in a separate PR to avoid holding up this one.